app/code/Magento/Sales/view/frontend/templates/email/items/creditmemo/default.phtml

```diff
@@ -16,7 +16,14 @@
                 <?php foreach ($block->getItemOptions() as $option) : ?>
                     <dt><strong><em><?= $block->escapeHtml($option['label']) ?></em></strong></dt>
                     <dd>
-                        <?= /* @noEscape */  nl2br($block->escapeHtml($option['value'])) ?>
+                        <?php
+                        if ($option['option_type'] == 'file') {
```

Comment:
Should be \Magento\Catalog\Api\Data\ProductCustomOptionInterface::OPTION_TYPE_FILE